sonar-core/src/test/java/org/sonar/core/persistence/DbTester.java

```diff
@@ -311,6 +310,42 @@ public void assertDbUnit(Class testClass, String filename, String... tables) {
     }
   }
 
+  public void assertColumnDefinition(String table, String column, int expectedType, @Nullable Integer expectedSize) {
+    try (Connection connection = openConnection();
+      PreparedStatement stmt = connection.prepareStatement("select * from " + table);
+      ResultSet res = stmt.executeQuery()) {
```

Comment:
Are you sure ? There are in the try-with-resources, so there are automatically closed